go.mod

```diff
@@ -0,0 +1,9 @@
+module developer-documentation
```

Comment:
This should be github.com/mattermost/mattermost-developer-documentation to avoid problems for importers of this module.